extern/region-4.18/src/cxcregion.h

```diff
@@ -1,7 +1,6 @@
 /*                                                                
-**  Copyright (C) 2007  Smithsonian Astrophysical Observatory 
+**  Copyright (C) 1999-2001,2004,2006-2007,2015-2016  Smithsonian Astrophysical Observatory 
```

Comment:
Is this copyright date correct too? CIAO 4.14 (presumably where `region-4.14` came from) was released in 2021 so any changes here are assumed to be later than that, aren't they?